packages/debian/metalk8s-sosreport-deb-2.4.0/debian/src/containerd.py

```diff
@@ -0,0 +1,73 @@
+#! /bin/env python
```

Comment:
so this file and the next are common to debian and redhat, they should reside in one place to ease maintenance and should be copied (linked) in build time.

Reply:
Is there no chance that these files could evolve with the time ?
If one file change for an OS family and not for another, it could be a problem if different OS families use the same.

Reply:
good point but I am not sure we want to support two codebases or have one file that supports both OSs, tbd

Reply:
Please, don't use `python`: use either `python2` or `python3`
Only use `python` when your code support both version.

I've had to patch too many scripts that were broken on ArchLinux because they now use Python 3 as default but people still think `python` == `python2` which is wrong (see [PEP-394](https://www.python.org/dev/peps/pep-0394/))

Reply:
As python3 will be the next standard and will be installed by default on debian systems, I suggest that we only use python3.